src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -75,6 +81,10 @@
 @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
 public class ChannelTabController extends AbstractChatTabController {
   private static final String USER_CSS_CLASS_FORMAT = "user-%s";
+  /**
+   * Taken from https://stackoverflow.com/questions/163360/regular-expression-to-match-urls-in-java
+   */
+  private static final Pattern URL_REGEX_PATTERN = Pattern.compile("^(https?|ftp|file)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]");
```

Comment:
Side note, py client has this stupid feature that allows people to post links to games by starting them with `faflive://` or `fafgame://`. Not advocating its return, but it's a cool piece of trivia.

Reply:
ehm how does this relate?

Reply:
Just wanted to mention it, because it's really useless and nice ^^